apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -661,7 +661,10 @@ export class DotEditContentHtmlService {
                     if (this.updateContentletInode) {
                         this.currentContentlet.inode = contentlet.inode;
                     }
-                    this.renderEditedContentlet(this.currentContentlet);
+                    // because: https://github.com/dotCMS/core/issues/21818
+                    setTimeout(() => {
+                        this.renderEditedContentlet(this.currentContentlet);
+                    }, 1800);
```

Comment:
We need to test that the setTimeout does not end in this issue: 

- https://github.com/dotCMS/core/issues/22210

Reply:
checked !